css/main.css

```diff
@@ -280,44 +465,54 @@ header .menu img {
   margin-top: 9rem;
 }
 
-.aboutme .description {
+.aboutme__description {
   color: var(--white);
   margin-top: 1rem;
+  line-height: 2.4rem;
 }
 
-.aboutme .aboutme_btn {
+.aboutme_btn {
   background-color: var(--orange);
   border: none;
   color: var(--white);
   padding: 1.5rem;
   margin-top: 3rem;
   font-size: 1.7rem;
-  width: 50%;
-  height: fit-content;
 }
 
-.aboutme .aboutme_btn_img .image {
-  width: 50%;
-}
+@media (min-width: 768px) {
+  .aboutme {
+    background-image: url(../images/illustration-aboutme-desktop_cut.svg);
+    background-position: 100% 10%;
+  }
+
+  .aboutme__content {
+    width: 115rem;
+    margin: auto;
+  }
 
-.aboutme .aboutme_btn_img .image img {
-  position: relative;
-  left: 30px;
+  .aboutme__description {
+    width: 51%;
+    font-size: 2rem;
+    line-height: 3.2rem;
+  }
 }
 
+/* -------skils------- */
+
 .skills {
-  margin-top: 12rem;
+  margin: 12rem 0;
 }
 
-.skills .skill {
+.skills__skill {
   display: flex;
   flex-direction: column;
   align-items: center;
   gap: 2.5rem;
   margin-bottom: 4rem;
 }
```

Comment:
- [ ] Kindly edit the text alignment of this section to make it match the Figma design. Also kindly adjust the background color of the sections to make it match Figma 💯 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/186997109-05c358a2-1f2b-4b6c-bbb7-89166a002462.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/186997158-7d005db5-4050-4e3b-bdac-cd6af9995fa2.png)